packages/components/src/Grid/Grid.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from "react";
+import classNames from "classnames";
+import { GridCell, GridCellProps } from "./GridCell";
+import styles from "./Grid.css";
+import alignments from "./GridAlign.css";
+
+interface GridProps {
+  /**
+   * Add spacing between elements.
+   */
+  readonly gap?: boolean;
+
+  /**
+   * Adjust the alignment of columns
```

Comment:
> would another element here be more intuitive?

Reply:
> would another element here be more intuitive?

Not really since it needs an element that follows the height of the parent. Used a flex as a workaround so it stretches out that card as well. Eventually, we'll have a Box component that makes it easy to do just that.

> I can see stretch being potentially confusing for devs

Tru, tho majority of the case, it'll either be start (default) or center. Rarely end (InputGroup uses end) or stretch. It's one of those if you know what it is, you'll appreciate it, but won't really even notice that it's there.

I'd leave it and explain further when the time comes.

Reply:
Added a link to align-items documentation